src/js/letters/components/Address.jsx

```diff
@@ -17,6 +19,10 @@ class Address extends React.Component {
     this.id = _.uniqueId('address-input-');
   }
 
+  componentDidMount() {
+    focusElement('select');
```

Comment:
Wonder if it might make more sense to have an "Edit address" header before this that we can focus on. I think generally speaking, this would be fine because to get here, they'd have to have clicked the "Edit" button, but if their address is returned as blank from the API, we automatically open this edit component. Focusing on the country field without any indication of what they're doing would be jarring, methinks.

I'll defer to @goldenmeanie on that one, though.

Reply:
Ah that's a great point, will see what @goldenmeanie thinks!

Reply:
Header sounds like a great idea @cvalarida @annekainicUSDS